applications/plugins/Compliant/CMakeLists.txt

```diff
@@ -251,6 +251,8 @@ if( SofaPython_FOUND )
         mapping/PythonMultiMapping.cpp
         python/_Compliant.cpp
         python/Binding_AssembledSystem.cpp
+
+        # note: need to fix assembled solver in python before removing
```

Comment:
Is there something that still to be done ?

Reply:
The code in this PR is loosely based on `Compliant/python/python.cpp`, so I have to update the code there to use this PR's code to avoid duplication. Unfortunately little time to do so.